dts/bindings/sensor/we,wsen-hids-common.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2022 Würth Elektronik eiSos GmbH & Co. KG
+# SPDX-License-Identifier: Apache-2.0
+
+properties:
+    drdy-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        Data-ready interrupt pin.
+
```

Comment:
Not sure if this empty line is intentional?

Reply:
Agreed; this empty line should be removed.

Reply:
Done.